frontend/models/people.js

```diff
@@ -12,9 +12,26 @@ function Person (rawData) {
   p.personClass = PersonClassService.getPersonClass({ id__eq: p.person_class })
   p.groups = GroupService.getGroups({ id__eq: p.groups })
   p.degrees = DegreeService.getDegrees({ id__eq: p.degrees })
+  p.RawData.formattedPersonName = p.RawData.FirstName + ' ' + p.RawData.LastName
+  p.RawData.personClass = p.personClass.Name
+
   return p
 }
 
-const PersonService = BaseService.constructDefaultService('api::person.person', 'person', 'people', Person)
+const peopleFilterFunctions = Object.assign({}, BaseService.filterFunctions)
+peopleFilterFunctions.searchNameAndClass = (key, value, obj) => obj.formattedPersonName.includes(value) || obj.FirstName.includes(value) || obj.personClass.includes(value)
```

Comment:
One thing we should do here for the searches is to force the search criteria to either upper case or lower case, and the value that we search should be either upper case or lower case (to match what we switched the criteria to).  This way, searching is not case specific.  (note, this need to be changed for the other search section as well.)

Example:

`obj.formattedPersonName.toLowerCase().includes(value.toLowerCase()) || obj.FirstName.toLowerCase().includes(value.toLowerCase()) || obj.personClass.toLowerCase().includes(value.toLowerCase())`